modules/phase_field/include/materials/OPInterfaceBarrierMaterial.h

```diff
@@ -0,0 +1,49 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef OPINTERFACEBARRIER_H
+#define OPINTERFACEBARRIER_H
```

Comment:
include guards should be names after the class name. I.e. `OPINTERFACEBARRIERMATERIAL_H`